src/styles/blocks/card.scss

```diff
@@ -0,0 +1,142 @@
+@import "../utils/variables.scss";
+
+.catalog {
+  padding-top: 50px;
+  display: grid;
+  grid-template-columns: repeat($column-count, $width-card);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (max-width: 487px) {
+    $column-count: 1;
+    grid-template-columns: repeat($column-count, $width-card);
+  }
```

Comment:
No need for this media query. Leave default property in catalog block as it is and the next media query will overwrite it